source/user/ci.rst

```diff
@@ -8,13 +8,14 @@ PySCF has separate molecular implementations for configuration
 interaction singles and doubles (CISD) (:mod:`ci`) and full
 configuration interaction (FCI) (:mod:`fci`). The functionalities of the
 CISD implementation are similar to the functionalities of MP2
-(:ref:`user_mp2`) and CCSD (:ref:`user_cc`) in PySCF.
+(:numref:`user_mp2`) and CCSD (:numref:`user_cc`) in PySCF.
 
 
 Introduction
 ============
 
-Configuration interaction is a post-Hartree--Fock method, diagonalising
+Configuration interaction (see e.g. :cite:`Knowles1984,SzaOst2012ci,Helgaker2000ci`)
+is a post-Hartree--Fock method, diagonalising
 the many-electron Hamiltonian matrix. FCI includes all Slater determinants of
```

Comment:
I think one should explicitly say here that the algorithm used in PySCF is the one in the Knowles paper, which should be cited in calculations employing the FCI module

Reply:
Also, as discussed on Slack, I think the three other papers could be cited here